thundra/integrations/botocore.py

```diff
@@ -378,81 +346,45 @@ def __init__(self):
 
     def get_operation_name(self, wrapped, instance, args, kwargs):
         _, request_data = args
-        return request_data.get('FunctionName', Constants.AWS_SERVICE_REQUEST)
+        return request_data.get('FunctionName', constants.AWS_SERVICE_REQUEST)
 
     def getRequestType(self, string):
-        if string in Constants.LambdaRequestType:
-            return Constants.LambdaRequestType[string]
-        return Constants.AWS_SERVICE_REQUEST
-
-    def inject_span_info(self, scope, wrapped, instance, args, kwargs, response,
-                         exception):
+        if string in constants.LambdaRequestType:
+            return constants.LambdaRequestType[string]
+        return constants.AWS_SERVICE_REQUEST
 
+    def before_call(self, scope, wrapped, instance, args, kwargs, response, exception):
         operation_name, request_data = args
-        self.lambdaFunction = request_data.get('FunctionName', Constants.AWS_SERVICE_REQUEST)
-        scope.span.domain_name = Constants.DomainNames['API']
-        scope.span.class_name = Constants.ClassNames['LAMBDA']
+        self.lambdaFunction = request_data.get('FunctionName', constants.AWS_SERVICE_REQUEST)
```

Comment:
Function name must be empty string instead of `AWS_SERVICE_REQUEST` constant if it is not available